test/test_cotrip.py

```diff
@@ -0,0 +1,561 @@
+def test_validate_incident_missing_required_field_description():
+  test_valid_output ={
+    "rtdh_timestamp": 1615866698.393723,
+    "rtdh_message_id": "dd962abd-0afa-4810-aac0-165edb834e71",
+    "event": {
+        "type": "Construction",
+        "sub_type": "Work Zone:roadway-relocation",
+        "source": {
+            "id": "349611",
+            "type": "Road Work",
+            "sub_type": "Road Construction",
+            "collection_timestamp": 1615482720
+        },
+        "lrs": None,
+        "geometry": "LINESTRING (-104.48011 37.007645, -104.480103 37.008034, -104.480125 37.008469, -104.480202 37.008904, -104.48024 37.009048, -104.480324 37.009338, -104.482475 37.015327, -104.482712 37.015945, -104.48288 37.016335, -104.482979 37.016521, -104.483208 37.016884, -104.483467 37.01722, -104.483612 37.01738, -104.483925 37.017681, -104.484253 37.017948, -104.484772 37.018295, -104.485138 37.01849, -104.485504 37.018661, -104.485886 37.01881, -104.486473 37.019005, -104.488014 37.019493)",
+        "header": {
+            "start_timestamp": 1615813200,
+            "end_timestamp": 1638255600
+        },
+        "detail": {       
+            "road_name": "I-25",
+            "road_number": "I-25",
+            "direction": "North"
+        },
+    }
+}
+  assert cotrip_translator.validate_alert(test_valid_output) == False
+
+def test_validate_alert_invalid_start_time():
+  test_valid_output = {
+    "rtdh_timestamp": 1615866698.393723,
+    "rtdh_message_id": "dd962abd-0afa-4810-aac0-165edb834e71",
+    "event": {
+        "type": "Construction",
+        "sub_type": "Work Zone:roadway-relocation",
+        "source": {
+            "id": "349611",
+            "type": "Road Work",
+            "sub_type": "Road Construction",
+            "collection_timestamp": 1615482720
+        },
+        "lrs": None,
+        "geometry": "LINESTRING (-104.48011 37.007645, -104.480103 37.008034, -104.480125 37.008469, -104.480202 37.008904, -104.48024 37.009048, -104.480324 37.009338, -104.482475 37.015327, -104.482712 37.015945, -104.48288 37.016335, -104.482979 37.016521, -104.483208 37.016884, -104.483467 37.01722, -104.483612 37.01738, -104.483925 37.017681, -104.484253 37.017948, -104.484772 37.018295, -104.485138 37.01849, -104.485504 37.018661, -104.485886 37.01881, -104.486473 37.019005, -104.488014 37.019493)",
+        "header": {
+            "description": "Road Construction - I-25 (Main St.) business loop from MP 1-2",
+            "start_timestamp": "1638S255600",
+            "end_timestamp": 1638255600
+        },
+        "detail": {       
+            "road_name": "I-25",
+            "road_number": "I-25",
+            "direction": "North"
+        },
+    }
+}
+  assert cotrip_translator.validate_alert(test_valid_output) == False
+
+def test_validate_alert_invalid():
+  test_valid_output = 'invalid output'
+  assert cotrip_translator.validate_alert(test_valid_output) == False
+
+def test_validate_incident_no_data():
```

Comment:
Should be named validate alert, not incident.

Reply:
should be fixed now